pages/profile.tsx

```diff
@@ -1,12 +1,28 @@
-import React, { useState } from "react";
+import React, { useEffect, useState } from "react";
 
 import { useAuth } from "../lib/authContext";
+import { useMutation, useQuery } from "@apollo/client";
 
 import ProfileGoalsSection from "../components/coding/ProfileGoalsSection";
 import UserProfileSection from "../components/coding/UserProfileSection";
+import BadgesSection from "../components/profile/BadgesSection";
+import {FETCH_CODING_BADGES, Data, IntroCourseUnit} from "../graphql/coding/userBadges/fetchUserBadges";
 
 export default function Profile(props) {
   const { user } = useAuth();
+  const { data } = useQuery(FETCH_CODING_BADGES, {
```

Comment:
can we move this fetch into the Badges Section component? instead of the profile?